index.js

```diff
@@ -1,44 +1,40 @@
+const os = require('os');
 const fs = require('fs');
 const path = require('path');
+const execFile = require('child_process').execFile;
 
-const exec = require('child_process').exec;
-const nodePath = process.execPath;
-const version = process.versions.v8;
-const tmpfile = path.join(__dirname, version+'.flags.json');
+const tmpfile = path.join(os.tmpdir(), process.versions.v8+'.flags.json');
 const exclusions = ['--help'];
 
 module.exports = function (cb) {
 
-  if (fs.existsSync(tmpfile)) {
-    process.nextTick(function () {
+  fs.exists(tmpfile, function (exists) {
+    try {
       cb(null, require(tmpfile))
```

Comment:
if the require fails, you want to `execFile` because the file doesn't exist.  I am pretty sure this is correct code